docs/eog/case-studies-cw20-timeline.rst

```diff
@@ -0,0 +1,145 @@
+   
+    - Collaborative notes documents with agenda and relevant links.
+    - Spreadsheets to disseminate the uCONFLY generated documents. 
+      
+  - Wrote documentation for the delivery team including
+   
+    - Zoom instructions 
+    - Duties roster
+    - Links index
+    - Start of day tasks for Event Lead/Chair
+    - Mini-workshop and demo session assignments
+      
+  - Made organiser slide deck.
+  
+Event run up actions
+  - Closed registration immediately upon decision to move event online.
+  - Issued full refunds to registrants who were no longer able to attend up to 7 days before the event start date.
+  - Issued partial refunds for the difference between the original priced and the new reduced price to registrants who still wanted to attend.
+  - Planned break-out groups distribution (sign up sheet for Discussion Groups session, randomly assigned for Collaborative Ideas session).
+  - Planned for video recording infrastructure (easy recording of sessions via Zoom).
+  - Had a Code of Conduct committee pre-event meeting.
+  - Constructed feedback form draft.
+
+
+Event Running Stage
+--------------------
+
+Month 0
+^^^^^^^^^^^^^^^^^^^^^^
+
+Ran event!
+
+Post event Stage
+--------------------
+
+Month 1 (April 2020)
+^^^^^^^^^^^^^^^^^^^^^^
+
+Agenda
+  - Updated the agenda with links to resources.
+  
+Feedback
+  - Emailed participants with a feedback form within a week after the event.
+  - Sent a reminder email a few days later.
+  - Held a feedback prize draw to thank them for taking the time.
+  
+Publicity
+  - Published a `blog post with highlights <https://software.ac.uk/blog/2020-04-21-highlights-collaborations-workshop-2020>`_ from the event.
+  - Published a `participant's experience of CW20 <https://software.ac.uk/blog/2020-04-29-ssi-collaborations-workshop-2020-remote-unconference-experience-and-notes>`_.
+  
+Resources
+  - Updated language of event website to show that it was in the past.
+  - Created a `Figshare portal <https://cw20.figshare.com/>`_ to share CW20 outputs
+  - Chased speakers for any un-captured resources.
+  - Collected Zoom recordings from the Hosts/Co-Hosts of each session.
+  - Prepared videos for editing by
+   
+    - Organising files.
+    - Updating processing instructions for Events Team.
+      
+  - Followed up with speed-blog authors and connected them with the Communications Team for publication to the SSI blog.
+
+
+Month 2 (May 2020)
+^^^^^^^^^^^^^^^^^^^^^^
+
+Publicity
+  - Published a reflections blog post about `how we moved the event online in three weeks <https://software.ac.uk/blog/2020-05-18-cw20-how-move-event-online-three-weeks>`_.
+  
+Resources
+  - 9/12 speed blogs published.
+
+Month 3 (June 2020)
+^^^^^^^^^^^^^^^^^^^^^^
+
+Publicity
+  - Announced availability of `Infrastructure resources <https://software.ac.uk/news/collaborations-workshop-2020-resources-now-available>`_.
+  
+Resources
+  - 10/12 speed blogs published.
```

Comment:
Is is the case that all 12 will be published - or were there 12 sessions - and 10 made it as published blogs of the right quality (at the moment it reads as more maybe coming).